ai-ml/jark-stack/terraform/addons.tf

```diff
@@ -399,3 +431,8 @@ data "aws_iam_policy_document" "karpenter_controller_policy" {
     sid       = "KarpenterControllerAdditionalPolicy"
   }
 }
+
+resource "aws_s3_bucket" "model_storage" {
```

Comment:
Let's add S3 VPC Endpoints in `vpc.tf`. This improves performance and reduces cost as traffic stays on AWS network.

Reply:
Let me address this in eks.tf with ECR and S3 endpoints.